specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2021-01-01/resourcemovercollection.json

```diff
@@ -13,6 +13,42 @@
     "application/json"
   ],
   "paths": {
+    "/providers/Microsoft.Migrate/operations/supportedResourceTypesForResourceMover": {
```

Comment:
Which RP will this API hit? This looks like an API that will hit global endpoint, as there is no idea of location associated here.

I see two options here:
a. If its a global endpoint, we can consider supporting this configuration across regional RP's, and use the azure traffic manager to route the traffic based on say "Geographic traffic-routing method".
https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-routing-methods#geographic-traffic-routing-method.
b. The other option is definitely, this to be an ARM api which is what our current proposal, however there might be a need for an wrapper if resource mover has to expose extra set of properties.